error: 4//tmp/tmpFumSHy/mbed_config.h: No such file or directory in import for SW4STM32

### Description
- Type: Bug 
- Related issue: `#abc`
- Priority: Blocker

---------------------------------------------------------------
## Bug

**Target**
All STM32 tried (F103, L151, F303)

**Toolchain:**
GCC_ARM

**Toolchain version:**
lastest updates in online mbed IDE, latest updates in SW4STM32

**Expected behavior**
build without error

**Actual behavior**
error during build  related to mbed_config.h

`14:17:51 **** Incremental Build of configuration Debug for project SX1280PingPong ****
make all 
Building file: ../SX1280Lib/sx1280-hal.cpp
Invoking: MCU G++ Compiler
...
arm-none-eabi-g++: error: 4//tmp/tmpFumSHy/mbed_config.h: No such file or directory
make: *** [SX1280Lib/sx1280-hal.o] Error 1`
 
**Steps to reproduce**
Import example project into online mbed IDE https://os.mbed.com/compiler/#import:/teams/Semtech/code/SX1280PingPong/
compile right clic on project / export program  select SW4STM32. Open project from SW4STM32 (file / open projects from filesystem), build  

The exact same project build correctly when exported in Keil uVision 5

--- a/tools/export/sw4stm32/__init__.py
+++ b/tools/export/sw4stm32/__init__.py
@@ -17,7 +17,7 @@ limitations under the License.
 from __future__ import print_function, absolute_import
 from builtins import str
 
-from os.path import splitext, basename, join
+from os.path import splitext, basename, relpath, join
 import shutil
 from tools.utils import mkdir
 from tools.export.gnuarmeclipse import GNUARMEclipse
@@ -290,7 +290,6 @@ class Sw4STM32(GNUARMEclipse):
         },
     }
 
-
     @classmethod
     def is_target_supported(cls, target_name):
         target = TARGET_MAP[target_name]
@@ -434,7 +433,10 @@ class Sw4STM32(GNUARMEclipse):
 
         self.resources.win_to_unix()
 
-        config_header = self.filter_dot(self.toolchain.get_config_header())
+        config_header = self.toolchain.get_config_header()
+        if config_header:
+            config_header = relpath(config_header, self.resources.file_basepath[config_header])
+        print('Config header: ' + config_header)
 
         libraries = []
         for lib in self.resources.libraries:

--- a/tools/export/sw4stm32/__init__.py
+++ b/tools/export/sw4stm32/__init__.py
@@ -436,7 +436,6 @@ class Sw4STM32(GNUARMEclipse):
         config_header = self.toolchain.get_config_header()
         if config_header:
             config_header = relpath(config_header, self.resources.file_basepath[config_header])
-        print('Config header: ' + config_header)
 
         libraries = []
         for lib in self.resources.libraries:

--- a/tools/export/sw4stm32/cproject_common.tmpl
+++ b/tools/export/sw4stm32/cproject_common.tmpl
@@ -28,7 +28,7 @@
 							<option id="fr.ac6.managedbuild.option.gnu.cross.floatabi.{{u.id}}" name="Floating-point ABI" superClass="fr.ac6.managedbuild.option.gnu.cross.floatabi" value="fr.ac6.managedbuild.option.gnu.cross.floatabi.{{opts['common']['arm.target.fpu.abi']}}" valueType="enumerated"/>
 							{% endif %}
 							<targetPlatform archList="all" binaryParser="org.eclipse.cdt.core.ELF" id="fr.ac6.managedbuild.targetPlatform.gnu.cross.{{u.id}}" isAbstract="false" osList="all" superClass="fr.ac6.managedbuild.targetPlatform.gnu.cross"/>
-							<builder buildPath="${workspace_loc:/{{name}}}/{{opts['name']}}" id="fr.ac6.managedbuild.builder.gnu.cross.{{u.id}}" keepEnvironmentInBuildfile="false" managedBuildOn="true" name="Gnu Make Builder" superClass="fr.ac6.managedbuild.builder.gnu.cross"/>
+							<builder buildPath="${workspace_loc:/{{name}}}/{{opts['name']}}" id="fr.ac6.managedbuild.builder.gnu.cross.{{u.id}}" keepEnvironmentInBuildfile="false" managedBuildOn="true" name="Gnu Make Builder" parallelBuildOn="true" parallelizationNumber="optimal" superClass="fr.ac6.managedbuild.builder.gnu.cross"/>
 							<tool id="fr.ac6.managedbuild.tool.gnu.cross.c.compiler.{{opts['uid']['tool_c_compiler']}}" name="MCU GCC Compiler" superClass="fr.ac6.managedbuild.tool.gnu.cross.c.compiler">
 								{% if cfg_id == 'debug' %}
 								<option id="fr.ac6.managedbuild.gnu.c.compiler.option.optimization.level.{{u.id}}" name="Optimization Level" superClass="fr.ac6.managedbuild.gnu.c.compiler.option.optimization.level" useByScannerDiscovery="false" value="fr.ac6.managedbuild.gnu.c.optimization.level.more" valueType="enumerated"/>

